pages/common/tldr.md

```diff
@@ -18,3 +18,7 @@
 - Update local pages (if the client supports caching):
 
 `tldr -u`
+
+- Render a local page for testing purposes (if the client supports it):
+
+`tldr --render {{path/to/page}}`
```

Comment:
This isn't a standard argument defined in the tldr [client specification](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#arguments), so unfortunately we can't include it here.

Reply:
Hey @sbrl, thanks for the feedback! Then i guess this PR is no longer useful... May it rest in peace 😄

Reply:
It most certainly will!  PRs, once closed/merged, seem to have a very peaceful existence here on GitHub! 😂

Reply:
@thomaswuensche sorry about that. We hope that you'll return with more contributions in the future :-)